hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/IncrementalContainerReportHandler.java

```diff
@@ -80,11 +80,16 @@ public void onMessage(final IncrementalContainerReportFromDatanode report,
         processContainerReplica(dd, replicaProto, publisher);
       } catch (ContainerNotFoundException e) {
         success = false;
-        LOG.warn("Container {} not found!", replicaProto.getContainerID());
+        LOG.warn("Container {} not found!",
+            replicaProto.getContainerID());
```

Comment:
@wycccccc This is just a formatting change. It isn't required here. Please avoid changing it, Apart from this changes Looks Good